app/src/main/java/org/xwiki/android/sync/activities/AccountListAdapter.kt

```diff
@@ -45,6 +41,15 @@ class AccountListAdapter(
             tvAccountServerAddress = view.findViewById(R.id.tvAccountServerAddress)
             llAccountItem = view.findViewById(R.id.llAccountItem)
         }
+
+        fun currentAccount(account: UserAccount, listener: AccountClickListener) {
```

Comment:
It is better to name like `setAccount`. Besides, you could rewrite it as:

```kotlin
var account: Account
    set(value) {
        field = value
        tvAccountName.text = value.accountName
        tvAccountServerAddress.text = value.serverAddress
    }
```

In this case you will be able to rewrite `init` with:

```kotlin
init {
    tvAccountName = view.findViewById(R.id.tvAccountName)
    tvAccountServerAddress = view.findViewById(R.id.tvAccountServerAddress)
    view.findViewById<LinearLayout>(R.id.llAccountItem).setOnClickListener {
        listener(account)
    }
}
```

Reply:
Can we keep it like setAccount() as function instead of property? I like it this way more.